packages/kolibri-common/composables/useCoachMetadataTags.js

```diff
@@ -0,0 +1,73 @@
+import { ref } from 'kolibri.lib.vueCompositionApi';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import coreStrings from 'kolibri.utils.coreStrings';
+import commonCoreStringsMixin from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+export function useCoachMetadataTags(contentNode) {
+  const tags = ref([]);
+
+  const getCategoryTags = () => {
+    if (!contentNode.categories) return [];
+    return contentNode.categories.map(category =>
+      commonCoreStringsMixin.methods.coreString(category),
```

Comment:
I think you should be able to import and use the `coreString` function directly

https://github.com/learningequality/kolibri/blob/b925b0485716dfe70e9c192f66dbe51837f6e0ad/kolibri/core/assets/src/mixins/commonCoreStrings.js#L1585-L1598

Reply:
There is currently some weirdness in the core JS API that means this is not as straight forward as it would appear. After the API has been refactored, this should be properly importable.